test/cases/regression-composer-works-behind-satellite.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+#
+# osbuild-composer can work with multiple subscriptions on the host system.
+# test this functionality by generating the certificates and creating
+# an image
```

Comment:
This is not precise.. It's simulating a custom repository with custom certificates and verifies that an image can be built using those. Check this commit https://github.com/osbuild/osbuild-composer/commit/2d75b25ffe1d34717b808dc2a85bd704ea9bd0a8